site/src/pages/WorkspacePage/WorkspaceTopbar.stories.tsx

```diff
@@ -266,7 +267,10 @@ export const WithQuota: Story = {
 	parameters: {
 		queries: [
 			{
-				key: getWorkspaceQuotaQueryKey(MockUser.username),
+				key: getWorkspaceQuotaQueryKey(
```

Comment:
What should be visible for the story WithQuota? this is what I am seeing.

<img width="919" alt="Screenshot 2024-09-11 at 4 55 18 PM" src="https://github.com/user-attachments/assets/95c56e89-7806-4ec1-a86c-6b587ba99be2">

Reply:
On `main` I do see 20 credits of 40 to the right of the workspace name

Reply:
No, there should be something there – I'm looking into this now, because it looks like the mocked Storybook query is failing for whatever reason